src/test/java/com/acrolinx/sidebar/pojo/settings/CheckOptionsTest.java

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2018-present Acrolinx GmbH */
+package com.acrolinx.sidebar.pojo.settings;
+
+import com.acrolinx.sidebar.pojo.document.IntRange;
+import com.acrolinx.sidebar.pojo.document.externalcontent.ExternalContent;
+import java.util.Collections;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class CheckOptionsTest
+{
+    @Test
+    void testToStringHasTheProperKeys()
+    {
+        CheckOptions checkOptions = getCheckOptions();
+
+        Assertions.assertEquals(
+                "{\"inputFormat\":\"AUTO\",\"requestDescription\":{\"documentReference\":\"foo\"},\"selection\":{\"ranges\":[[0,0]]},\"externalContent\":{}}",
+                checkOptions.toString());
+    }
+
+    @Test
+    void testToStringFailsIfNoValidKeysProvided()
```

Comment:
I don't understand the purpose of this test method. We create a CheckOptions instances via the same helper method as in the first test method, so what scenario is this one covering that the first one does not already cover?